sunpy/net/jsoc/jsoc.py

```diff
@@ -522,13 +523,8 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         dl_set = True
         if not downloader:
             dl_set = False
-            downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=max_conn)
-
-        if downloader.max_conn * kwargs['max_splits'] > 10:
-            warn_user("JSOC does not support more than 10 parallel connections. " +
-                      f"Changing the number of parallel connections to {2 * self.default_max_conn}.")
-            kwargs['max_splits'] = 2
-            downloader.max_conn = self.default_max_conn
+            # Private communication from JSOC say we should not use more than one connection.
+            downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=1)
```

Comment:
While this change is good in isolation, it has no effect on queries passed down from a Fido search.

Reply:
So what should I do?